src/server/page.ts

```diff
@@ -191,13 +196,8 @@ export class Page extends SdkObject {
       this._setIsError(error);
     }
     this._browserContext.emit(BrowserContext.Events.Page, this);
-    const openerDelegate = this._delegate.openerDelegate();
-    if (openerDelegate) {
-      openerDelegate.pageOrError().then(openerPage => {
-        if (openerPage instanceof Page && !openerPage.isClosed())
-          openerPage.emit(Page.Events.Popup, this);
-      });
-    }
+    if (this.isClosed())
```

Comment:
Let's add a comment explaining this one.

Reply:
Done.